Dockerfile

```diff
@@ -1,47 +1,58 @@
 # syntax = docker/dockerfile:1.0-experimental
 ARG VAGRANT_VERSION=2.2.19
 
-
-FROM ubuntu:bionic as base
-
-RUN apt update \
-    && apt install -y --no-install-recommends \
-        bash \
-        ca-certificates \
-        curl \
-        git \
-        gosu \
-        kmod \
-        libvirt-bin \
-        openssh-client \
-        qemu-utils \
-        rsync \
-    && rm -rf /var/lib/apt/lists \
+FROM debian:stable-slim as base
+
+RUN apt-get -y -qq update \
+    && apt-get -y --no-install-recommends install \
+      bash \
+      ca-certificates \
+      curl \
+      git \
+      gosu \
+      kmod \
+      libguestfs-tools \
+      libvirt0 \
+      libvirt-clients \
+      libvirt-dev \
+      libxml2-dev \
+      libxslt-dev \
```

Comment:
The dev packages are installed as part of the build layer, including them as part of the base will bloat the slim image.